### PR TITLE
feat: add DepositData SSZ container

### LINKED ISSUE #52: Add SSZ container: `DepositData`
Follow these instructions: https://github.com/lambdaclass/lambda_ethereum_consensus/blob/main/native/ssz_nif/README.md

#### `DepositData`

```python
class DepositData(Container):
    pubkey: BLSPubkey
    withdrawal_credentials: Bytes32
    amount: Gwei
    signature: BLSSignature  # Signing over DepositMessage
```

### PR DESCRIPTION
**Pull request by karasakalmt**
**Description**

This issue is to adding the DepositData SSZ container

Closes #52

### CODE REVIEW

File: native/libp2p_nif/go.sum [hunk 1]
```diff
@@ -10,6 +10,8 @@ git.apache.org/thrift.git v0.0.0-20180902110319-2566ecd5d999/go.mod h1:fPE2ZNJGy
 github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03qcyfWMU=
```
Comment (mpaulucci) [inline native/libp2p_nif/go.sum:10]: there shouldn't be any changes to this file

**Comment by mpaulucci** [comment]
looking good so far! Plz sync with main

File: native/libp2p_nif/go.sum (hunk 1 above)
Comment (karasakalmt) [reply native/libp2p_nif/go.sum:10]: it happened when I run the `make spec-test` actually, I will revert to the old `go.sum` file

**Comment by karasakalmt** [comment]
except from last line it seems like it is good to go actually it is the same line but idk why it seems like it is changed @mpaulucci

**Review by mpaulucci — APPROVED** [review]

**Comment by phklive** [comment]
@all-contributors please add @karasakalmt for code

**Pull request merged**